script.js

```diff
@@ -1,12 +1,236 @@
 //You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+async function  setup() {
+  // const allEpisodes = getAllEpisodes();
+  const rootElem = document.getElementById("root");
+  const navBar = document.getElementById("navbar")
+  const input = document.createElement("input")
+  navBar.appendChild(input)
+  const pCount = document.createElement("p")
+  navBar.appendChild(pCount)
+  const select = document.getElementById("select")
+
+
+  function titleCodeGenerator(episode){
+    let seasonCode = (episode.season < 10) ? '0' + episode.season : episode.season;
+    let episodeCode = (episode.number < 10) ? '0' + episode.number : episode.number;
+    return `S${seasonCode}E${episodeCode}`;
+  }
```

Comment:
Its great to use of ternary if statement, it makes your code short and readable.